trashtalk/views/cleanups.py

```diff
@@ -52,13 +50,14 @@ def get(cleanup_id):
     if current_user.is_authenticated:
         bool_participated = check_participants(current_user.id, cleanups.participants)
     else:
-        bool_participated = False #Default to anonymous users not being participants
+        bool_participated = False  # Default to anonymous users not being participants
 
-    map = current_app.config['GOOGLE_MAPS_ENDPOINT'].format(cleanups.gmap_query, current_app.config['GOOGLE_MAPS_KEY'])
+    gmap = "{0}{1}".format(current_app.config['GOOGLE_MAPS_ENDPOINT'],
+                           cleanups.gmap_query)
```

Comment:
If **address** really has that many benefits over **coordinates** isn't possible to just take **address** as a whole from **geocode** results. Is it worth it to break it down and then reconstruct it?

Reply:
Also, is it possible we're using different types of APIs? Because I still can't get your version to work.

Reply:
Geopy makes sure we have a valid address by looking it up for us. This keeps users from entering things like "MY MOTHERS HOUSE" or nothing at all.